dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -66,6 +66,10 @@ BlobStore::BlobStore(String storage_name, const FileProviderPtr & file_provider_
     , blob_stats(log, delegator, config_)
     , cached_files(config.cached_fd_size)
 {
+    if (config.block_alignment_bytes % 4096 != 0)
```

Comment:
4k is too waste of space IMO. Since this param should be adjusted by developers, I think we don't need a warning here.

Reply:
removed and port to config